sounddevice.py

```diff
@@ -2413,7 +2413,7 @@ def __init__(self, channel_map=None, change_device_parameters=False,
 
 class WasapiSettings:
 
-    def __init__(self, exclusive=False):
+    def __init__(self, exclusive=False, auto_convert=True):
```

Comment:
Why is this on by default?

In general, I try not to change PortAudio's defaults unless there is a good reason to do so. Is there?

In fact, this might be very confusing, because passing `sd.WasapiSettings()` as `extra_settings` to a stream would change its behavior, while passing `sd.WasapiSettings(auto_convert=False)` would not.

I would expect `extra_settings=sd.WasapiSettings()` to have no effect at all.

Or am I missing something?